### DISCUSSION TITLE
⭐️ Mantine Figma Component Library ⭐️

### DISCUSSION THREAD

**Discussion by elgreco84** [mantinedev/mantine#5053] [Show]
### Introducing Pretine
The Figma Library for Mantine

https://www.figma.com/community/file/1293978471602433537/pretine-the-mantine-ui-kit-library

In an effort to support and enhance the Mantine community, the [Ravn](https://www.ravn.co) design team has created a well organized, modern Figma library. We hope this library will encourage even more people to use Mantine as they build web applications. We hope to continue improving and updating the library over time. We welcome feedback and suggestions and we will do our best to incorporate them into Pretine. Enjoy!

**Comment by devalirzayev** [comment]
Wow! Thanks a lot 🙌
